basics/Strings examples/find function multiple occurance.py

```diff
@@ -0,0 +1,17 @@
+str = input("Type the string:")
+ch = input("Type the character/word you want to find the occurance:")
+start=0
+n=len(str)
+count=set()
+for i in range(0,n+1):

```

Comment:
This finds `asa` from `asasa` twice, even though after the first `asa`, only `sa` is left and that does not contain another `asa`. Is this what you wanted it to do? This might be a good "fix this code" exercise.

Reply:
Ok..